### PR TITLE
Set/Get -AppxPackageAutoUpdateSettings PowerShell cmdlets

### PR DESCRIPTION
**Pull request by RDMacLachlan**
Two new Cmdlets being made available.

### CODE REVIEW

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (dariomws) [inline docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:14]:
```suggestion
Available in the Windows Insider Preview Builds of Windows 10, is the `Set-AppxPackageAutoUpdateSettings` PowerShell cmdlet. The `Set-AppxPackageAutoUpdateSettings` PowerShell cmdlet provides access to configure a specific Windows App's Auto Update and Repair settings. Including pausing update checks.
```

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (dariomws) [inline docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:43]:
```suggestion
This cmdlet will update the Auto Update settings of the *PackageName_8h66172c634n0* Windows App to target an AppInstaller file on a network accessible file share every two hours, displaying a prompt to the user. Allowing for the Windows App to update to any version (higher or lower) despite the version of the installed Windows App.
```

Comment (dariomws) [reply docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:43]: Let´s use the same syntax through the article for Auto Update and Repair settings

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (dariomws) [inline docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:45]:
```suggestion
### Example 2: Disable the Auto Repair setting for an App
```

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (dariomws) [inline docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:47]:
```suggestion
PS C:\> Set-AppxPackageAutoUpdateSettings -AppInstallerUri https://website.com/PackageName.appinstaller -PackageFamilyName PackageName_8h66172c634n0 -DisableAutoRepairs
```

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (dariomws) [inline docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:347]:
```suggestion
[Get-AppxPackage](Get-AppxPackage.md)

[Get-AppxPackageManifest](Get-AppxPackageManifest.md)

[Move-AppxPackage](Move-AppxPackage.md)

[Remove-AppxPackage](Remove-AppxPackage.md)
```

File: docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md:13]:
```suggestion
The `Get-AppxPackageAutoUpdateSettings` PowerShell cmdlet is available in the Windows Insider Preview Builds of Windows 10. The `Get-AppxPackageAutoUpdateSettings` PowerShell cmdlet provides visibility to the settings configured on a Windows 10 client device for a particular Windows App. 
```

File: docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md:32]:
```suggestion
This will return the Auto Update and Repair settings registered to the user for all configured and installed Windows Apps on the device.
```

File: docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md:34]:
```suggestion
### Example 2: Get App Package Auto Update Settings for all users
```

File: docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md:39]:
```suggestion
This will return the Auto Update and Repair settings for all configured and installed Windows Apps that have been registered for all users.
```

File: docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md:41]:
```suggestion
### Example 3: Get a single App Package Auto Update setting
```

File: docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md:46]:
```suggestion
This will return the Auto Update and Repair settings for a specific Windows App that has been installed and registered to the signed-in user.
```

File: docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md:51]:
```suggestion
Specifies the Package Full Name of the app that is being queried.
```

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:14]:
```suggestion
Available in the Windows Insider Preview Builds of windows 10, is the `Set-AppxPackageAutoUpdateSettings` PowerShell cmdlet. The `Set-AppxPackageAutoUpdateSettings` PowerShell cmdlet provides access to configure a specific Windows App's Auto Update and Repair settings. Including pausing update checks.
```
```suggestion
The `Get-AppxPackageAutoUpdateSettings` PowerShell cmdlet is available in the Windows Insider Preview Builds of Windows 10. The `Set-AppxPackageAutoUpdateSettings` PowerShell cmdlet provides access to configure a specific Windows App's Auto Update and Repair settings. Including pausing update checks.
```

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:38]:
```suggestion
### Example 1: Update the Auto Update settings for an app
```

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:43]:
```suggestion
This cmdlet will update the Auto Update settings of the *PackageName_8h66172c634n0* Windows App to target an AppInstaller file on a network accessible file share every two hours, displaying a prompt to the user and allowing for the Windows App to update to any version (higher or lower) despite the version of the installed Windows App.
```

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:69]:
```suggestion
Specifies the duration of time in hours that the Windows App will not check for updates.
```

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:189]:
```suggestion
Specifies that the next update of the Windows App can be of a higher or lower version.
```

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:294]:
```suggestion
Specifies that an override can be applied to the Developer Set settings.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RDMacLachlan, please commit the suggested changes of both @dariomws and myself if you agree with them. Many thanks.

**Review by tseward — APPROVED** [review]
Approved once @dariomws changes are implemented.

File: docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md
Comment (RDMacLachlan) [reply docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md:32]: When we add ("Install") an application, we both stage and register the app. The settings are applied to the app. Without the use of the "-allusers" parameter, the returned results are only for Windows Apps registered for the users, and with auto update and repair configured settings.

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (RDMacLachlan) [reply docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:14]: Should this comment of it being available only in the Insider Preview not be included?

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (RDMacLachlan) [reply docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:294]: I'm not sure this should be capitalized. Perhaps the word "configured" would be a better fit?

**Comment by RDMacLachlan** [comment]
This Doc's article can't go live until after June 24th.

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [reply docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:14]: I have changed my suggestion slightly.

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [reply docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:294]: Thank you for the comment. I've changed my suggestion to suit.

File: docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Set-AppxPackageAutoUpdateSettings.md:14]:
```suggestion
The `Get-AppxPackageAutoUpdateSettings` PowerShell cmdlet is only available in the Windows Insider Preview Builds of Windows 10. The `Set-AppxPackageAutoUpdateSettings` PowerShell cmdlet provides access to configure a specific Windows App's Auto Update and Repair settings. Including pausing update checks.
```

File: docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/appx/Get-AppxPackageAutoUpdateSettings.md:32]:
```suggestion
This will return the Auto Update and Repair settings for all configured and installed Windows Apps on the device, and registered to the user.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
@RDMacLachlan - many thanks for your feedback. I have made the necessary changes to the suggestions. Could you please commit these 5 remaining suggested changes?

**Comment by dariomws** [comment]
> This Doc's article can't go live until after June 24th.

@RDMacLachlan please change it to draft until that date. To convert a pull request to a draft, look for the “Convert to draft” link in the Reviewers section on the pull request’s Conversations tab. Clicking the link displays a modal asking you to confirm the change.

**Comment by RDMacLachlan** [comment]
> @RDMacLachlan please change it to draft until that date. To convert a pull request to a draft, look for the “Convert to draft” link in the Reviewers section on the pull request’s Conversations tab. Clicking the link displays a modal asking you to confirm the change.
> 
> If it is still only available in the Insider Preview on June 24th, I think you should keep that information.

Thank you @dariomws . I've updated the status to be a Draft. I'll update the status on the 24th.

**Comment by RDMacLachlan** [comment]
> @RDMacLachlan - many thanks for your feedback. I have made the necessary changes to the suggestions. Could you please commit these 5 remaining suggested changes?

Thank you for the edits @JohanFreelancer9 . I have accepted all of your recommended changes. Appreciate the review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by dariomws** [comment]
@Kateyanne could you kindly add the "do not merge" label here? Thank you!

**Comment by RDMacLachlan** [comment]
Switched this to be no longer in draft. @Kateyanne  and @dariomws can we remove the "Do Not Merge" label so that these articles can be merged :-).

**Pull request merged**
